saltproc/depcode.py

```diff
@@ -314,12 +327,12 @@ def get_tra_or_dec(self, input_file):
 
         Parameters
         -----------
-        input_file: str
+        input_file : str
             Serpent2 input file name and path.
 
         Returns
         --------
```

Comment:
There is one extra "-" on the `Parameters` and the `Returns` sections

Reply:
Nice catch